tests/v2/validation/provisioning/provisioning_node_driver_linux_test.go

```diff
@@ -0,0 +1,203 @@
+package provisioning
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/machinepools"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (r *RKE2NodeDriverProvisioningTestSuite) setupLinux() {
+	r.SetupSuiteLinuxOnly()
+}
+
+func (r *RKE2NodeDriverProvisioningTestSuite) ProvisioningRKE2ClusterLinuxOnly(provider Provider) {
+	subSession := r.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := r.client.WithSession(subSession)
+	require.NoError(r.T(), err)
+
+	cloudCredential, err := provider.CloudCredFunc(client)
+	require.NoError(r.T(), err)
+
+	allNodeRolesLinux := []map[string]bool{
+		{
+			"controlplane": true,
+			"etcd":         true,
+			"worker":       true,
```

Comment:
This is what I'm referring to with the linuxWorker key over in `tests/framework/extensions/machinepools/machinepools.go`

Reply:
linuxWorker was a remnant of a prior approach. I have switched it back to worker